pgx-utils/src/rewriter.rs

```diff
@@ -482,7 +449,7 @@ impl PgGuardRewriter {
             func.sig.ident.span(),
         );
 
-        let arg_list = PgGuardRewriter::build_arg_list(&sig, false);
+        let (arg_list, _matchable) = PgGuardRewriter::build_arg_list(&sig, false);
```

Comment:
This function had its return signature altered but the latter half of the tuple is used in only one location? Is there another way to do this?

Reply:
I'm not sure it's entirely correct that I'm ignoring it in this case. I'm not totally clear on how `PgGuardRewriter .item_fn` is called and for what reasons. It looks like it's usually called from [here](https://github.com/tcdi/pgx/blob/develop/pgx-macros/src/lib.rs#L592-L598), where `rewrite_args` is always supplied as `false`, but is also called from [here](https://github.com/tcdi/pgx/blob/develop/pgx-macros/src/lib.rs#L46), where it's also hard-coded to a value of `false`. So based on I think it might be okay to entirely remove `item_fn_with_rewrite`, the argument `rewrite_args`, and so on. I didn't want to do that surgery without understanding more, though.

Reply:
Oops - I misread. In the first case, it's got a value of `true` hard-coded, not `false`. Not sure how I managed that one...